### ISSUE TITLE
Defaults for new multi-site subsites

### ISSUE THREAD

**Issue by joelworsham** [realbigplugins/client-dash#160]

**Comment by d4mation** [comment]
I just fielded a ticket that I think was related to this. 

Is this Issue for having the ability to have Default Menu Customizations Multisite-wide that the sub-sites can override? 

Would this work by assuming no Plugins active for Customizing on the Network Dashboard (Showing a sub-site dashboard since they're different)? That way they can do basic things like renaming Dashboard or whatever and have all sub-sites inherit it for that Role and then the sub-site can make their own Customizations on top of that?

**Comment by joelworsham** [comment]
Not sure of the logistics. CD in general needs multi-site support. I envision network activating it, then from the network admin they can SOMEHOW setup defaults? Not sure, but could be nice.

**Comment by d4mation** [comment]
Yeah, I was playing with it while replying to the ticket. Seems that Client Dash works within a Multisite (Network activation works, activation per-site works, ect. with no apparent problems), but doesn't gain any special Multisite features (such as defaults for sub-sites).

This along with #159 seem like they'd be great things to implement in the next minor version.
